coreneuron/io/reports/report_handler.cpp

```diff
@@ -51,7 +51,7 @@ void ReportHandler::create_report(double dt, double tstop, double delay) {
         }
         if (!vars_to_report.empty()) {
             auto report_event =
-                std::make_unique<ReportEvent>(dt, t, vars_to_report, m_report_config.output_path);
+                std::make_unique<ReportEvent>(dt, t, vars_to_report, m_report_config.output_path.data());
```

Comment:
See line 14, and replace in the other cases if needed.

Reply:
same!